policyengine_taxsim/config/variable_mappings.yaml

```diff
@@ -413,25 +413,43 @@ policyengine_to_taxsim:
     group_order: 4
     full_text_group: "State Tax Calculation"
     group_column: 1
-    special_cases:
+    pre_simulation:
       - ms:
-          implemented: false
-          variable: na_pe
+          implemented: true
```

Comment:
does this only split the standard deduction variable?

Reply:
yes this pre_simulation field applies to those particular states during {state}_standard_deduction simulation, first it checks the state matched or not, then executes {state}_files_separately and based on the result it applies from variables _indiv or _joint.

Reply:
It should not just be splitting the standard deduction but multiple other variables to get accurate results

Reply:
Other notable programs that follow this structure are:
itemized deductions
state income tax before non refundable credits 
taxable income 

And selected tax credits

Reply:
1. sum them up when multiple output is produced
2. for tests files keep them as separated